<ui version="4.0" >
 <class>HdrGLWidgetControl</class>
 <widget class="QWidget" name="HdrGLWidgetControl" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>282</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QCheckBox" name="enableHdr" >
     <property name="text" >
      <string>Enable HDR</string>
     </property>
     <property name="checked" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="bloom" >
        <property name="text" >
         <string>Bloom</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QWidget" native="1" name="widget_2" >
          <layout class="QGridLayout" name="gridLayout_3" >
           <item row="0" column="0" colspan="2" >
            <widget class="QCheckBox" name="bloomAfterTonemapping" >
             <property name="text" >
              <string>Bloom after tonemapping</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_4" >
             <property name="text" >
              <string>Downsize:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QSlider" name="bloomDownsize" >
             <property name="maximum" >
              <number>5</number>
             </property>
             <property name="pageStep" >
              <number>1</number>
             </property>
             <property name="value" >
              <number>1</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_5" >
             <property name="text" >
              <string>Strength:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QDoubleSpinBox" name="bloomStrength" >
             <property name="maximum" >
              <double>10.000000000000000</double>
             </property>
             <property name="singleStep" >
              <double>0.100000000000000</double>
             </property>
             <property name="value" >
              <double>1.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="exposureAuto" >
        <property name="text" >
         <string>Automatic exposure adjustment</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" >
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QStackedWidget" name="exposureOptions" >
          <property name="currentIndex" >
           <number>0</number>
          </property>
          <widget class="QWidget" name="page" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>210</width>
             <height>72</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout_2" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Exposure:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="exposure" >
              <property name="minimum" >
               <double>0.010000000000000</double>
              </property>
              <property name="maximum" >
               <double>50.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.200000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <spacer name="verticalSpacer" >
              <property name="orientation" >
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>20</width>
                <height>27</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2" >
           <property name="geometry" >
            <rect>
             <x>0</x>
             <y>0</y>
             <width>210</width>
             <height>72</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>Speed:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="exposureAutoSpeed" >
              <property name="minimum" >
               <double>0.010000000000000</double>
              </property>
              <property name="maximum" >
               <double>10.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Target:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDoubleSpinBox" name="exposureAutoTarget" >
              <property name="minimum" >
               <double>0.010000000000000</double>
              </property>
              <property name="maximum" >
               <double>50.000000000000000</double>
              </property>
              <property name="singleStep" >
               <double>0.100000000000000</double>
              </property>
              <property name="value" >
               <double>1.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enableHdr</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>87</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel" >
     <x>129</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bloom</sender>
   <signal>toggled(bool)</signal>
   <receiver>widget_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel" >
     <x>79</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
